Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM php:7.2-fpm
+
+# Keep composer happy
+ENV COMPOSER_ALLOW_SUPERUSER 1
+# Keep nullmailer from pestering for a name
+ENV DEBIAN_FRONTEND noninteractive
+
+EXPOSE 80
+
+WORKDIR /
+
+RUN apt-get update \
+  && apt-get install -y gnupg \
+  && curl -sL https://deb.nodesource.com/setup_9.x | bash - \
+  && echo 'deb http://apt.newrelic.com/debian/ newrelic non-free' > /etc/apt/sources.list.d/newrelic.list \
+  && curl -fsSL https://download.newrelic.com/548C16BF.gpg | apt-key add - \
+  && echo 'deb http://nginx.org/packages/mainline/debian/ stretch nginx deb-src http://nginx.org/packages/mainline/debian/ stretch nginx' > /etc/apt/sources.list.d/nginx.list \
+  && curl -fsSL http://nginx.org/keys/nginx_signing.key | apt-key add - \
+  && apt-get update \
+  && apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y install \
+  fuse \
+  git \
+  gnupg \
+  libffi-dev \
+  mariadb-client \
+  newrelic-php5 \
```

Comment:
Does it matter here that we are running PHP7 not 5?

Reply:
It doesn't.  This is their standard package.  Technically, isn't 7 actually 5.7?

Reply:
Oh and this is the package we're using now, and we run 7.1